bridge/engine/ActiveMediaList.h

```diff
@@ -160,6 +163,7 @@ class ActiveMediaList
         int32_t _nonZeroLevelsShortWindow;
         float _maxRecentLevel;
         float _noiseLevel;
+        bool _is_ptt;
```

Comment:
isPtt

Reply:
Shall we remove all `_` prefixes here? it's a struct, members are not private.